### ISSUE TITLE
[Question] Alternative Authentication possible?

### ISSUE THREAD

**Issue by noxify** [BKWLD/decoy#47]
Hi,

i would ask, if it's possible to use an alternative authentication package?

Is it "just" changing the guard and the policy class in the config or do i have do change something else?

Thanks!

**Comment by noxify** [comment]
I tested it a bit and it seems that it requires more than just changing the config.

I have used rinvex/fort which extends the default auth with some good features.

For me, Decoy seems to be the solution to speed up my development, because it includes everything what i need.

My idea/plan is/was to allow my users to control their posts via the decoy control panel.
In my first prototype i created everything from scratch... i don't really like it....

With the curent implementation i don't see the possibility to allow this access without syncing the users table with the admins table.

But correct me if i'm wrong :)


Anyway... thanks a lot for this awesome work :)

**Comment by weotch** [comment]
We have done this in the past, replacing the Decoy auth wtih a client's SSO solution.  But it's been awhile and since we only added tests recently, I'm not surprised that's not working as intended.  Originally we used the [Sentry package](https://github.com/cartalyst/sentry) but [replaced it](https://github.com/BKWLD/decoy/commit/95a863135b625c758c65bacc35fe155603381d40) as it grew in a direction we weren't interested in.

I do _want_ replacing the auth to be possible and would accept a PR if you find the things that need changing to make it work again.  But if it's not working now, I don't see us taking on this work ourselves until we have a client of our own that requires it.
